agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.h

```diff
@@ -287,6 +287,22 @@ class backhaul_manager : public btl::transport_socket_thread {
     bool get_neighbor_links(const sMacAddr &neighbor_mac_filter,
                             std::map<std::string, std::vector<sMacAddr>> &neighbor_links_map);
 
+    /*
+     * @brief List of known 1905 neighbor devices
+     * 
+     * key:     1905.1 device AL-MAC
+     * value:   s1905NeighborDevice
+     * Devices are being added to the list when receiving a 1905.1 Topology Discovery message from
+     * an unknown 1905.1 device. Every 1905.1 device shall send this message every 60 seconds, and
+     * we update the time stamp in which the message is received.
+     */
+    struct s1905NeighborDevice {
+        std::chrono::steady_clock::time_point last_seen;
+        sMacAddr local_iface_mac;
+        bool bridge_exist; // Not sure from where to take this information
```

Comment:
I mean the field "Existence of IEEE 802.1 bridges" which is on "1905.1 neighbor device TLV" on IEEE 1905.1-2013 Table 6-15.
We need to send this TLV in the Topology Response message.

Reply:
So that's the first case. It needs to be detected using the LLDP protocol. And as mentioned, this has to be per local interface (I wrote per interface pair but that was wrong).

Reply:
Ok, I will change the comment about it on a fixup.
Not implementing it as part of this PR. Out of scope.